### DISCUSSION TITLE
Code Review and Process

### DISCUSSION THREAD

**Discussion by baetheus** [baetheus/fun#155] [General]
@baoshan and @pixeleet I just realized that I started working through the 2.0 stuff without asking if you guys wanted to be involved! Are either of you interested in picking up some of the work? Admittedly, most of it is pretty slog-like. Formatting, checking for missing implementations, writing tests, and slow slow slow documentation. But if you're interested let me know how much you'd like (or just assign stuff to yourself if you want) and I'll pass it your way.

Additionally, do you guys have any interest in doing code reviews on the PRs that I am making? Right no I'm pretty confident of tests and coverage picking up most errors but extra eyes are always good. Are either of you interested in doing reviews? If so I'll stop merging after the github actions pass and wait for a review.

Thanks for the contributions you guys have already made and I hope to see more from both of you :D

**Comment by pixeleet** [comment]
Hey man, it's great to see action again around the project!
You're going super fast with the improvements for 2.0, kudos!

Besides RTE/SRTE happy to chime in with getting 2.0 out of course.
I'll have a bit of time available in the coming months, happy to help where
I can

Brandon Blaylock ***@***.***> ezt írta (időpont: 2022. szept.
17., Szo, 21:59):

**Reply by baetheus** [reply]
Heya @pixeleet, are you still interested in implementing FnAsyncEither and StateFnAsyncEither?

**Comment by pixeleet** [comment]
Hey Brandon,

Great to hear from you buddy, how is the merry potter life? :)

I’m currently mostly using effect-ts, needed to go back to node land for a moment.

I am no longer pursuing FAE in it’s current form.

I would be very interested though in implementing a platform-deno for Effect
https://github.com/effect-ts/platform￼

Drop me a line somewhere if you wanna catch up :)
